website/docs/CI Starter Workflows/Nodejs-Starter-Workflow.md

```diff
@@ -0,0 +1,129 @@
+# Nodejs Starter Workflow Sample
+
+## Node Example
+
+This guide covers configuring continuous integration pipelines for a Node starter workflow in Harness CI. If you’re new to Harness CI please read our Onboarding Guide and Nodejs Sample Pipeline Build Configuration Guides first.
```

Comment:
```suggestion
This guide covers configuring continuous integration pipelines for a Node starter workflow in Harness CI. If you’re new to Harness CI, start with Onboarding Guide and Nodejs Sample Pipeline Build Configuration Guides.
```

Reply:
@codewdhruv hyperlink `Onboarding Guide` and `Nodejs Sample Pipeline Build Configuration` Guides.